diesel/src/associations/mod.rs

```diff
@@ -47,24 +46,19 @@
 //! # }
 //! ```
 //!
-//! Note that in addition to the `#[has_many]` and `#[belongs_to]` annotations, we also need to
+//! Note that in addition to the `#[belongs_to]` annotation, we also need to
 //! `#[derive(Associations)]`
 //!
-//! `#[has_many]` should be passed the name of the table that the children will be loaded from,
-//! while `#[belongs_to]` is given the name of the struct that represents the parent. Both types
+//! `#[belongs_to]` is given the name of the struct that represents the parent. Both types
```

Comment:
"Both types" means "the annotated struct and the parent", right? Maybe write that out?